baselines/dom.generated.d.ts

```diff
@@ -3448,7 +3448,11 @@ declare var Document: {
     new(): Document;
 }
 
-interface DocumentFragment extends Node, NodeSelector, ParentNode {
+interface DocumentFragment extends Node {
+    querySelector<K extends keyof ElementTagNameMap>(selectors: K): ElementTagNameMap[K] | null;
```

Comment:
As I specified in https://github.com/Microsoft/TypeScript/issues/14246, NodeSelector's properties are defined with ParentNode, not NodeSelector. So we must merge and remove NodeSelector by this pr and #201.